system/security/digisign/digisign.cpp

```diff
@@ -266,6 +266,7 @@ static std::once_flag dsmInitFlag;
 
 MODULE_INIT(INIT_PRIORITY_STANDARD)
 {
+    OpenSSL_add_all_algorithms();
```

Comment:
Should this be wrapped in -
```
#ifdef _USE_OPENSSL 
#endif
```

Reply:
Could this be in the createDigitalSignatureManagerInstance*() wrappers with a std::call_once ?
It seems ok here but MODULE_INIT is called always, even if we never call any of these routines.

Reply:
Agreed